### PR TITLE
enable sentinel bind usage with multi ip

### PR DESCRIPTION
**Pull request by ragingdave**
#### Pull Request (PR) description
There is currently an issue where if you use the allowed array of IP for the sentinel bind, this fails and the template becomes something completely broken as the resulting template shows:
```
bind ["10.0.0.155", "127.0.0.1"]
```

This PR fixes that by using the same technique in the base redis usage to force it into an array and flatten it to generate a correct bind line.

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
The type also allows undef and this ends up with an empty `bind` statement and breaks the tests. Please have a look at that.

File: manifests/sentinel.pp [hunk 1]
```diff
@@ -149,6 +149,8 @@
     }
   }
 
+  $sentinel_bind_arr = [$sentinel_bind].flatten
```
Comment (igalic) [inline manifests/sentinel.pp:152]: see delete_undef_values for @ekohl's comment, or it's documentation for how to do that with filter

**Review by igalic — CHANGES_REQUESTED** [review]

**Comment by bastelfreak** [comment]
@ragingdave thanks for the PR! Can you add a unit test for this change?

**Review by bastelfreak — APPROVED** [review]

File: manifests/sentinel.pp [hunk 2]
```diff
@@ -149,6 +149,8 @@
     }
   }
 
+  $sentinel_bind_arr = delete_undef_values([$sentinel_bind]).flatten
```
Comment (igalic) [inline manifests/sentinel.pp:152]: it might make more sense to first `flatten()` and then `delete_undef_values()`

**Pull request merged**